extensions/sql-database-projects/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "sql-database-projects",
+  "displayName": "SQL Server Database Projects",
+  "description": "SQL Server Database Projects for Azure Data Studio designing and deploying database schemas.",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "engines": {
+		"vscode": "^1.30.1",
+		"azdata": ">=1.12.0"
+  },
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/extensions/import/Microsoft_SQL_Server_Import_Extension_and_Tools_Import_Flat_File_Preview.docx",
+  "icon": "images/sqlserver.png",
+  "aiKey": "AIF-c5594e2d-38b5-4d3b-ab1b-ed5d4fe8ee40",
+  "activationEvents": [
+    "*"
+  ],
+  "main": "./out/main",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/azuredatastudio.git"
+  },
+  "extensionDependencies": [
+    "Microsoft.mssql"
+  ],
+  "contributes": {
+    "commands": [
+      {
+        "command": "sqlDatabaseProjects.new",
+        "title": "%sqlDatabaseProjects.new%",
+        "category": "%sqlDatabaseProjects.displayName%"
+      },
+      {
+        "command": "sqlDatabaseProjects.open",
+        "title": "%sqlDatabaseProjects.open%",
+        "category": "%sqlDatabaseProjects.displayName%"
+      }
+    ],
+    "menus": {
+        "commandPalette": [
+          {
+            "command": "sqlDatabaseProjects.new"
+          },
+          {
+            "command": "sqlDatabaseProjects.open"
+          }
+        ]
+    }
+  },
+  "dependencies": {
+    "vscode-extension-telemetry": "0.0.18",
+    "vscode-nls": "^3.2.1"
+  },
+  "devDependencies": {
+    "tslint": "^5.8.0",
+    "typescript": "^2.6.1",
+    "azdata": "^1.0.0"
+  }
```

Comment:
You can remove azdata and vscode-extension-telemetry since you're not using either of them currently